domains/admin-portal/AdminPortal.Web/AdminPortal.Tests/Controllers/WhitelistedOrganizationsControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using AdminPortal._Features_;
+using AdminPortal.Dtos.Response;
+using AdminPortal.Tests.Setup;
+using Microsoft.AspNetCore.Mvc.Testing;
+using NSubstitute;
+
+namespace AdminPortal.Tests.Controllers;
+
+public class WhitelistedOrganizationsControllerTests
```

Comment:
What will happen here if we do changes in authorization api? would it break these tests? If so, i think the test should be in authorization and let the client here be as stupid as possible and just make sure the api contracts is good